src/main/java/net/dv8tion/jda/api/interactions/callbacks/IAutoCompleteCallback.java

```diff
@@ -46,14 +46,13 @@ public interface IAutoCompleteCallback extends Interaction
      *         The choice suggestions to present to the user, 0-{@value OptionData#MAX_CHOICES} choices
      *
      * @throws IllegalArgumentException
-     *         If any of the following is true:
      *         <ul>
-     *             <li>{@code null} is provided</li>
-     *             <li>More than {@value OptionData#MAX_CHOICES} are added</li>
-     *             <li>Any of the choice names is empty or longer than {@value OptionData#MAX_CHOICE_NAME_LENGTH}</li>
-     *             <li>The option type is incompatible with the choice type</li>
-     *             <li>The numeric value of any of the choices is not between {@value  OptionData#MIN_NEGATIVE_NUMBER} and {@value OptionData#MAX_POSITIVE_NUMBER}</li>
-     *             <li>The string value of any of the choices is empty or longer than {@value OptionData#MAX_CHOICE_VALUE_LENGTH}</li>
+     *             <li>If {@code null} is provided</li>
+     *             <li>If more than {@value OptionData#MAX_CHOICES} are added</li>
```

Comment:
```suggestion
     *             <li>If more than {@value OptionData#MAX_CHOICES} choices are added</li>
```
Not really related to this PR, but thought I'd review it regardless.
Personally find 
> If more than 25 choices are added

better than
> If more than 25 are added